pkg/virt-controller/watch/migration.go

```diff
@@ -429,6 +429,15 @@ func (c *MigrationController) updateStatus(migration *virtv1.VirtualMachineInsta
 		}
 	}
 
+	migrationPhaseArgs := &processMigrationPhaseArgs{
+		Pod:                 pod,
+		AttachmentPod:       attachmentPod,
+		PodExists:           podExists,
+		AttachmentPodExists: attachmentPodExists,
+		ConditionManager:    conditionManager,
+		VmiConditionManager: vmiConditionManager,
+	}
+
```

Comment:
Can we create this struct right before calling the `processMigrationPhase` function? IMHO, it is not clear why this struct is needed here.